config/default.js

```diff
@@ -175,4 +176,10 @@ module.exports = {
   fxaConfig: null,
 
   proxyEnabled: false,
+
+  // If true, enable a route that explicitly triggers a server error
+  // to test our internal error handler.
+  allowErrorSimulation: false,
```

Comment:
For the test routes for error pages (`/404`, `/500`, etc.) I just checked for `isDevelopment`. If the user is in development mode AND added a `SENTRY_DSN` environment variable I'd say that's enough to say they want error simulation and the extra config is just more work/docs/etc.